docs/gathered-data.md

```diff
@@ -13,1040 +13,1878 @@ archives are packaged as `.tar.gz` files in `/var/lib/insights-operator`.
+- 4.9.27+
 
-* Location in archive: config/pod/openshift-kube-controller-manager/logs/{pod-name}/errors.log
-* Since versions:
-  - 4.9.27
-  - 4.10.6
-  - 4.11+
+### Changes
+None
 
 
 ## LogsOfNamespace
 
-creates a gathering closure which collects logs from pods in the provided namespace
+Creates a gathering closure which collects logs from pods in the provided namespace.
+
 Params is of type GatherLogsOfNamespaceParams:
-  - namespace string - namespace from which to collect logs
-  - tail_lines int64 - a number of log lines to keep for each container
+- namespace string - namespace from which to collect logs
+- tail_lines int64 - a number of log lines to keep for each container
 
-The Kubernetes API:
+### API Reference
+- https://github.com/kubernetes/client-go/blob/master/kubernetes/typed/core/v1/pod_expansion.go#L48
+- https://docs.openshift.com/container-platform/4.6/rest_api/workloads_apis/pod-core-v1.html#apiv1namespacesnamespacepodsnamelog
 
-	https://github.com/kubernetes/client-go/blob/master/kubernetes/typed/core/v1/pod_expansion.go#L48
+### Sample data
+- [docs/insights-archive-sample/conditional/namespaces/openshift-cluster-samples-operator/pods/cluster-samples-operator-8ffb9b45f-49mjr/containers/cluster-samples-operator/logs/last-100-lines.log](./insights-archive-sample/conditional/namespaces/openshift-cluster-samples-operator/pods/cluster-samples-operator-8ffb9b45f-49mjr/containers/cluster-samples-operator/logs/last-100-lines.log)
+
+### Location in archive
+- `conditional/namespaces/{namespace}/pods/{pod_name}/containers/{container_name}/logs/last-{n}-lines.log`
+
+### Config ID
+`conditional/logs_of_namespace`
 
-Response see:
+### Released version
+- 4.9.0
 
-	https://docs.openshift.com/container-platform/4.6/rest_api/workloads_apis/pod-core-v1.html#apiv1namespacesnamespacepodsnamelog
+### Backported versions
+None
 
-* Location in archive: conditional/namespaces/{namespace}/pods/{pod_name}/containers/{container_name}/logs/last-{n}-lines.log
-* Id in config: conditional/logs_of_namespace
-* Since versions:
-  - 4.9+
+### Changes
+None
 
 
 ## MachineAutoscalers
 
-collects MachineAutoscalers definition
+Collects MachineAutoscalers definition.
 
-The Kubernetes api:
+### API Reference
+- https://github.com/openshift/cluster-autoscaler-operator/blob/master/pkg/apis/autoscaling/v1beta1/machineautoscaler_types.go
+- https://docs.openshift.com/container-platform/4.7/rest_api/autoscale_apis/machineautoscaler-autoscaling-openshift-io-v1beta1.html#machineautoscaler-autoscaling-openshift-io-v1beta1
+
+### Sample data
+- [docs/insights-archive-sample/config/machineautoscalers/openshift-machine-api/worker-us-east-1a.json](./insights-archive-sample/config/machineautoscalers/openshift-machine-api/worker-us-east-1a.json)
 
-	https://github.com/openshift/cluster-autoscaler-operator/blob/master/pkg/apis/autoscaling/v1beta1/machineautoscaler_types.go
+### Location in archive
+- `config/machineautoscalers/{namespace}/{resource}.json`
```

Comment:
Does `{resource}` mean the resource kind, resource name, resource UUID or something else?